packages/scrollbar/__tests__/scrollbar.spec.ts

```diff
@@ -50,6 +50,34 @@ describe('ScrollBar', () => {
     expect(wrapper.find('.is-horizontal div').attributes('style')).toContain('width: 40%; transform: translateX(150%); webkit-transform: translateX(150%)')
   })
 
+  test('both vertical and horizontal', async () => {
+    const outerHeight = 200
+    const innerHeight = 500
+    const outerWidth = 200
+    const innerWidth = 500
+    const wrapper = _mount(`
+      <el-scrollbar style="height: ${outerHeight}px; width: ${outerWidth}px;">
+        <div style="height: ${innerHeight}px; width: ${innerWidth}px;"></div>
+      </el-scrollbar>
+    `)
+
+    const scrollDom = wrapper.find('.el-scrollbar__wrap').element
+
+    defineGetter(scrollDom, 'clientHeight', outerHeight)
+    defineGetter(scrollDom, 'scrollHeight', innerHeight)
+    defineGetter(scrollDom, 'clientWidth', outerWidth)
+    defineGetter(scrollDom, 'scrollWidth', innerWidth)
```

Comment:
Restore the getter after you hijacked it. Using the return function returned by `defineGetter` to restore it.

Reply:
The hijack happened on the target's prototype, we need to restore it in order to prevent contaminating the prototype chain.